src/quickstart.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -euo pipefail
+
+OWNER=microshift-io
+REPO=microshift
+BOOTC_ARCHIVE=""
```

Comment:
Shall we prepend `_` to indicate this is "private" implementation detail?
```suggestion
_BOOTC_ARCHIVE=""
```